src/bencode.rs

```diff
@@ -1936,6 +1936,7 @@ mod bench {
     use self::test::Bencher;
 
     use std::io;
+    use std::iter;
```

Comment:
`range` is already in the prelude, `use` is not necessary. Use `range` directly.